src/components/ha-date-range-picker.ts

```diff
@@ -36,11 +37,14 @@ export class HaDateRangePicker extends LitElement {
 
   @property({ type: Boolean }) private _hour24format = false;
 
+  @property({ type: String }) private _rtlDirection = "ltr";
```

Comment:
let's not call this `rtlDirection` as its value isn't a boolean, let's call it `_textDirection` or `_direction` or `_dir`